sql/updates/world/master/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,19 @@
+-- Shift existing rows by 3 bits to reserve space for 3 other tables
+UPDATE `waypoint_data` SET `id`=(`id` << 3) ORDER BY `id` DESC;
+UPDATE `creature_addon` SET `path_id`=(`path_id` << 3);
```

Comment:
also referenced by event_scripts, spell_scripts and waypoint_scripts where command = 20 (datalong) and 35 (dataint && datalong == 2)
as well as all c++ scripts using `MotionMaster::MovePath(uint32, bool)`

Reply:
adjusted, thanks!